### DISCUSSION TITLE
Question about opus

### DISCUSSION THREAD

**Discussion by Uktltat** [enzo1982/freac#655] [Help & Support]
Does the fre:ac 1.1.7 build, from October 24 for Windows uses the opus 1.5.2 encoder?

**Comment by SatanicMechanic** [comment]
Current release is from 2023 and includes 1.3.1. hopefully we get a new build soon.

**Comment by SatanicMechanic** [comment]
If you don't mind running unstable, that has current (1.5.2) right now. https://github.com/enzo1982/freac/releases/tag/continuous
